database/src/pool.rs

```diff
@@ -183,6 +183,21 @@ pub trait Connection: Send + Sync {
     /// Add an item to the `benchmark_requests`, if the `benchmark_request`
     /// exists it will be ignored
     async fn insert_benchmark_request(&self, benchmark_request: &BenchmarkRequest);
+
+    /// Gets the benchmark requests matching the status. Optionally provide the
+    /// number of days from whence to search from
+    async fn get_benchmark_requests_by_status(
```

Comment:
I know that the old code didn't uphold this, but I'd rather avoid unwraps in new code, and explicitly return `Result`s everywhere. Especially since the cron job is a background task, and if it panics, the whole system grinds to a halt. We might want to add some auto-restarting functionality for it, just in case.

Reply:
I've used anyhow; ac7b0ff7e3787f8c1b3a00f022f85a57a1094f17